app/components/post_draft/send_handler/index.ts

```diff
@@ -10,20 +10,24 @@ import {General, Permissions} from '@constants';
 import {MAX_MESSAGE_LENGTH_FALLBACK} from '@constants/post_draft';
 import {observeChannel, observeChannelInfo, observeCurrentChannel} from '@queries/servers/channel';
 import {queryAllCustomEmojis} from '@queries/servers/custom_emoji';
+import {queryDraft} from '@queries/servers/drafts';
 import {observePermissionForChannel} from '@queries/servers/role';
 import {observeConfigBooleanValue, observeConfigIntValue, observeCurrentUserId} from '@queries/servers/system';
 import {observeUser} from '@queries/servers/user';
 
-import SendHandler from './send_handler';
+import SendHandler, {INITIAL_PRIORITY} from './send_handler';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
+import type DraftModel from '@typings/database/models/servers/draft';
 
 type OwnProps = {
     rootId: string;
     channelId: string;
     channelIsArchived?: boolean;
 }
 
+const observeFirst = (v: DraftModel[]) => v[0]?.observe() || of$(undefined);
```

Comment:
this is duplicate here app/components/post_draft/index.ts can you improve that? perhaps move it to queries/server/draft and call it `observeFirstDraft` or something